### DISCUSSION TITLE
Support Map like data structure?

### DISCUSSION THREAD

**Discussion by wentao** [fzyzcjy/flutter_rust_bridge#843] [Ideas]
I wonder if flutter_rust_bridge has any plan to support Map like data structure, similar to how uniffi-rs supports it.

https://mozilla.github.io/uniffi-rs/udl/builtin_types.html

**Comment by fzyzcjy** [comment]
Sure, it should be quite trivial to implement it: Because we already support `Vec`, so `Map` can be delegated to `Vec` for implementation. feel free to PR!
